### ISSUE TITLE
Help me.

### ISSUE THREAD

**Issue by stars66** [wubicheng/DWVR#2]
Hi wubicheng, I'm interested in your project, Can you contact me and discuss more? Thanks

**Comment by wubicheng** [comment]
So, what can I help you?

**Comment by stars66** [comment]
Thanks for your reply, do yo have a skype? My id: ska1227@163.com

**Comment by wubicheng** [comment]
Sorry, I don't hava skype account, and my spoken english is very bad, so I can't use english to talk with you directly. If you need any help, you can leave messages to me in here or send an e-mail directly. My e-mail is 371295004@qq.com.
Finally, I come from China, can you tell me where are you from?

**Comment by stars66** [comment]
Hi Are you a developer of "Card King"?

**Comment by wubicheng** [comment]
No, I think I am not the developer of "Card King", I'm just the developer of its Chinese version, but now this version has been paused, and this online project is my little experiment.
And now I think maybe I shouldn't upload the project to the public repository, because the project should be secret. I thought people wouldn't search for this project, so I use this repository as my version management system, but it turned out that I was wrong. At first, I thought you wanted to work together to work on this project, so I didn't rush to delete the repository, but I think it might not be good to keep the repository existing. And I don't know why you delete your communication way, but it makes me alert, so I will delete my e-mail too. 
If I misunderstand you, I'm sorry, and you can continue to contact me with my e-mail. I hope you don't spread your local project out, this may cause me trouble.
